ixmp/core.py

```diff
@@ -286,6 +286,31 @@ def add_region_synonym(self, region, mapped_to):
 
         self._backend.set_node(region, synonym=mapped_to)
 
+    def timeslices(self):
+        """Return all subannual time slices defined for the IAMC-style
```

Comment:
Please follow the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) standard: the short summary should fit on one line.